docs/react/guides/ssr.md

```diff
@@ -0,0 +1,162 @@
+---
+outline: deep
+---
+
+# SSR
+
+Wagmi uses client-only external stores (such as localStorage and `mipd`) to show the user the most relevant data as quickly as possible on first render.
+
+However, the caveat of using these external client stores is that frameworks which incorporate SSR (such as Next.js) will throw hydration warnings on the client when it identifies mismatches between the server-rendered HTML and the client-rendered HTML.
+
+To stop this from happening, you can toggle on the `ssr` property in the Wagmi Config.
+
+```tsx
+import { createConfig, http } from 'wagmi'
+import { mainnet, sepolia } from 'wagmi/chains'
+
+const config = createConfig({ // [!code focus:99]
+  chains: [mainnet, sepolia],
+  ssr: true, // [!code ++]
+  transports: {
+    [mainnet.id]: http(),
+    [sepolia.id]: http(),
+  },
+})
+```
+
+Turning on the `ssr` property means that content from the external stores will be hydrated on the client after the initial mount.
+
+## Persistence using Cookies
+
+As a result of turning on the `ssr` property, external persistent stores like `localStorage` will be hydrated on the client **after the initial mount**.
+
+This means that you will still see a flash of "empty" data on the client (ie. a `"disconnected"` account instead of a `"reconnecting"` account, or an empty address instead of the last connected address) until after the first mount, when the store hydrates.
```

Comment:
Cool, what about adding a "loading" state? e.g. to prevent clicking on the connect button before data gets hydrated (although that should usually be pretty quick)

Reply:
Wouldn't this be handled by checking `status` in `useAccount`?